### ISSUE TITLE
NoneType object has no attribute find_all

### ISSUE THREAD

**Issue by ryantm** [makefu/hydra-check#13]
I think I'm using the latest hydra-check available on nixpkgs master, and this happens:

```
$ hydra-check boxfort                                                                                         
Build Status for nixpkgs.boxfort.x86_64-linux on unstable                                                                               
✔ boxfort-unstable-2019-09-19 from 2020-06-26 - https://hydra.nixos.org/build/123247390  

$ hydra-check boxfort --channel=master --arch=aarch64-linux --short
Traceback (most recent call last):
  File "/nix/store/d9xc2p69yyyi3v8sa6k0a9jgaik1jn0i-python3.7-hydra-check-1.1.1/bin/.hydra-check-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/d9xc2p69yyyi3v8sa6k0a9jgaik1jn0i-python3.7-hydra-check-1.1.1/lib/python3.7/site-packages/hydracheck/cli.py", line 129, in main
    builds = list(parse_build_html(resp))
  File "/nix/store/d9xc2p69yyyi3v8sa6k0a9jgaik1jn0i-python3.7-hydra-check-1.1.1/lib/python3.7/site-packages/hydracheck/cli.py", line 73, in parse_build_html
    for row in doc.find("tbody").find_all("tr"):
AttributeError: 'NoneType' object has no attribute 'find_all'
```

Am I doing something wrong?

**Comment by ryantm** [comment]
Looking into this more, it appears there is Hydra job for boxfort for aarch64-linux.

**Comment by makefu** [comment]
```
~ hydra-check boxfort --channel=master --arch=aarch64-linux --short --url 
https://hydra.nixos.org/job/nixpkgs/trunk/boxfort.aarch64-linux
```
https://hydra.nixos.org/job/nixpkgs/trunk/boxfort.aarch64-linux -> `This job is not a member of the latest evaluation of its jobset. This means it was removed or had an evaluation error.`
not sure what is wrong there but there does not seem to be a list of results. for this specific case we could create a better error message, however i do not even know then this would happen.

**Comment by doronbehar** [comment]
I get this error with `hydra-check openjdk14`.

**Comment by makefu** [comment]
@doronbehar it looks fine for me, maybe it is related to some hydra `weird` behavior?
```
hydra-check openjdk14
Build Status for nixpkgs.openjdk14.x86_64-linux on unstable
✖ (Failed) openjdk-14.0.1-ga from 2020-07-25 - https://hydra.nixos.org/build/124473491

Last Builds:
✖ (Failed) openjdk-14.0.1-ga from 2020-07-18 - https://hydra.nixos.org/build/124101086
✖ (Failed) openjdk-14.0.1-ga from 2020-07-13 - https://hydra.nixos.org/build/123900315
✔ openjdk-14.0.1-ga from 2020-06-26 - https://hydra.nixos.org/build/123240385
✔ openjdk-14.0.1-ga from 2020-06-20 - https://hydra.nixos.org/build/122665659
✔ openjdk-14.0.1-ga from 2020-06-19 - https://hydra.nixos.org/build/122624126
✔ openjdk-14.0.1-ga from 2020-06-15 - https://hydra.nixos.org/build/122263745
✔ openjdk-14.0.1-ga from 2020-06-11 - https://hydra.nixos.org/build/122087693
✖ (Failed) openjdk-14.0.1-ga from 2020-06-10 - https://hydra.nixos.org/build/121803653
✔ openjdk-14.0.1-ga from 2020-05-29 - https://hydra.nixos.org/build/120816206
```

**Comment by symphorien** [comment]
I get it with
```
$  hydra-check mirage-im --arch=x86_64-darwin
Traceback (most recent call last):
  File "/nix/store/wcm7lcphbka64r05vvkw1f66jqv1zih1-python3.8-hydra-check-1.1.1/bin/.hydra-check-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/wcm7lcphbka64r05vvkw1f66jqv1zih1-python3.8-hydra-check-1.1.1/lib/python3.8/site-packages/hydracheck/cli.py", line 129, in main
    builds = list(parse_build_html(resp))
  File "/nix/store/wcm7lcphbka64r05vvkw1f66jqv1zih1-python3.8-hydra-check-1.1.1/lib/python3.8/site-packages/hydracheck/cli.py", line 73, in parse_build_html
    for row in doc.find("tbody").find_all("tr"):
AttributeError: 'NoneType' object has no attribute 'find_all'
```
and any non existing name:
```
$  hydra-check nonexisting
Traceback (most recent call last):
  File "/nix/store/wcm7lcphbka64r05vvkw1f66jqv1zih1-python3.8-hydra-check-1.1.1/bin/.hydra-check-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/wcm7lcphbka64r05vvkw1f66jqv1zih1-python3.8-hydra-check-1.1.1/lib/python3.8/site-packages/hydracheck/cli.py", line 129, in main
    builds = list(parse_build_html(resp))
  File "/nix/store/wcm7lcphbka64r05vvkw1f66jqv1zih1-python3.8-hydra-check-1.1.1/lib/python3.8/site-packages/hydracheck/cli.py", line 73, in parse_build_html
    for row in doc.find("tbody").find_all("tr"):
AttributeError: 'NoneType' object has no attribute 'find_all'
```

**Comment by makefu** [comment]
fixed with `1.2.0`, hydra-check will not fail if a package does not exist or is not eval'ing

However we are limited to what hydra tells:

```
$ hydra-check nonexisting hello
Build Status for nixpkgs.nonexisting.x86_64-linux on unstable
⚠ This job is not a member of the latest evaluation of its jobset. This means it was removed or had an evaluation error.
Build Status for nixpkgs.hello.x86_64-linux on unstable
✔ hello-2.10 from 2020-12-11 - https://hydra.nixos.org/build/132718189
```

hydra-check will print the alert on the page https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.nonexisting.x86_64-linux

![image](https://user-images.githubusercontent.com/115218/103324231-e67c3d00-4a46-11eb-8641-51fa55c0fbab.png)
